### PR TITLE
i18n(fr): Update mardown-content.mdx

### PR DESCRIPTION
**Pull request by pioupia**
#### What kind of changes does this PR include?

- Translated content

#### Description

- What does this PR change? Give us a brief description.
I updated the translation of the page `mardown-content.mdx`

### CODE REVIEW

**Comment by pioupia** [comment]
Depends on :
1. `layouts.mdx`

File: src/content/docs/fr/guides/markdown-content.mdx [hunk 1]
```diff
@@ -123,13 +149,50 @@ const {frontmatter} = Astro.props;
 </html>
 ```
 
-📚 En savoir plus à propos des [mises en page Markdown](/fr/core-concepts/layouts/#mises-en-page-markdownmdx).
+📚 En savoir plus à propos des [Layouts Markdown](/fr/core-concepts/layouts/#markdownmdx-layouts).
```
Comment (sarah11918) [inline src/content/docs/fr/guides/markdown-content.mdx:152]:
```suggestion
📚 En savoir plus à propos des [Layouts Markdown](/fr/core-concepts/layouts/#mises-en-page-markdownmdx).
```

File: src/content/docs/fr/guides/markdown-content.mdx [hunk 2]
```diff
@@ -89,9 +115,9 @@ Vous pouvez aussi passer le flag `--drafts` lors de l'exécution de `astro build
 
 Astro fournit des fonctionnalités Markdown supplémentaires et intégrées disponibles lors de l'utilisation de fichiers Markdown et MDX.
 
-### Le frontmatter `layout`
+### L'avant-propos `layout`
 
-Astro fourni aux pages Markdown et MDX une propriété spéciale `layout` frontmatter qui peut spécifier un chemin relatif (ou [alias](/fr/guides/aliases/)) à un [composant de mise en page Astro](/fr/core-concepts/layouts/#mises-en-page-markdownmdx).
+Astro fourni aux pages Markdown et MDX une propriété spéciale `layout` d'avant-propos qui peut spécifier un chemin relatif (ou [alias](/fr/guides/aliases/)) à un [composant layout Astro](/fr/core-concepts/layouts/#markdownmdx-layouts).
```
Comment (sarah11918) [inline src/content/docs/fr/guides/markdown-content.mdx:120]:
```suggestion
Astro fourni aux pages Markdown et MDX une propriété spéciale `layout` d'avant-propos qui peut spécifier un chemin relatif (ou [alias](/fr/guides/aliases/)) à un [composant layout Astro](/fr/core-concepts/layouts/#mises-en-page-markdownmdx).
```

**Review by sarah11918 — COMMENTED** [review]
Just tidied up some of the updated links. (Also can't make the suggestion inline here, but line 420: `#le-frontmatter-layout` needs to become `#lavant-propos-layout` to match the new heading here. I can fix that after/outside of this review!

File: src/content/docs/fr/guides/markdown-content.mdx [hunk 3]
```diff
@@ -1,12 +1,39 @@
...
+Avec [l'intégration @astrojs/mdx](/fr/guides/integrations-guide/mdx/) installée, Astro supporte aussi les fichiers [MDX](https://mdxjs.com/) (`.mdx`) qui apportent des fonctionnalités supplémentaires, telles que la prise en charge des expressions et des composants JavaScript dans votre contenu Markdown.
+
+Utilisez l'un ou les deux types de fichiers pour écrire du contenu Markdown !
 
 ## Pages Markdown et MDX
 
+### Collections de contenu
+
+Vous pouvez gérer vos fichiers Markdown et MDX dans Astro dans un dossier spécial `src/content/`. Les [collections de contenu](/fr/guides/content-collections/) vous aident à organiser votre contenu, à valider vos frontmatter et fournir un type-safety TypeScript automatique lorsque que vous travaillez sur votre contenu.
+You can manage your Markdown and MDX files in Astro in a special `src/content/` folder. [Content collections](/fr/guides/content-collections/) help you organize your content, validate your frontmatter, and provide automatic TypeScript type-safety while working with your content.
```
Comment (Yan-Thomas) [inline src/content/docs/fr/guides/markdown-content.mdx:20]: oops, English text
```suggestion
```

File: src/content/docs/fr/guides/markdown-content.mdx [hunk 4]
```diff
@@ -116,275 +144,190 @@ const {frontmatter} = Astro.props;
...
-  <body>
-    <!-- Ajoutez d'autres composants UI ici, comme des navigations et pieds de page communs. -->
-    <h1>{content.title} par {content.author}</h1>
-    <!-- 2. Le HTML généré sera passé dans le Slot par défaut. -->
-    <slot />
-    <p>Écrit le : {content.date}</p>
-  </body>
-</html>
-```
+Il ajoute aussi des fonctionnalités supplémentaires au MDX standard, notamment la prise en charge du [frontmatter de style Markdown dans MDX] (https://mdxjs.com/guides/frontmatter/). Cela vous permet d'utiliser la plupart des fonctionnalités Markdown intégrées d'Astro, comme la propriété de [frontmatter `layout`](#frontmatter-layout) et un paramètre pour les [pages brouillons](#pages-brouillons).
```
Comment (Yan-Thomas) [inline src/content/docs/fr/guides/markdown-content.mdx:187]: Small link issue here
```suggestion
Il ajoute aussi des fonctionnalités supplémentaires au MDX standard, notamment la prise en charge du [frontmatter de style Markdown dans MDX](https://mdxjs.com/guides/frontmatter/). Cela vous permet d'utiliser la plupart des fonctionnalités Markdown intégrées d'Astro, comme la propriété de [frontmatter `layout`](#frontmatter-layout) et un paramètre pour les [pages brouillons](#pages-brouillons).
```

**Review by Yan-Thomas — APPROVED** [review]
LGTM, thanks @pioupia! I commited a fix for the small issues I found, hope you don't mind. 🙌 💜

**Comment by pioupia** [comment]
> LGTM, thanks @pioupia! I commited a fix for the small issues I found, hope you don't mind. raised_hands purple_heart

Thank you very much! Sorry, I didn't see these concerns because of the fatigue. Thanks for the update!

**Pull request merged**